src/cli/commands.js

```diff
@@ -2,92 +2,108 @@
 'use strict';
 
 module.exports = {
-  oc: {
+  usage: 'Usage: $0 <command> [options]',
+  commands: {
 
     dev: {
-      help: 'Runs a local oc test registry in order to develop and test components',
+      cmd: 'dev <dirPath> [port] [baseUrl]',
+      example: {
+        cmd: '$0 dev ../all-components 3001 127.0.0.1:3001 --fallbackRegistryUrl=http://anotherhost:anotherport/'
+      },
+      description: 'Runs a local oc test registry in order to develop and test components',
       options: {
-        dirName: {
-          help: 'The name of the directory to watch, where the components are stored'
-        },
-        port: {
-          help: 'The port where to start a local oc instance. Default 3000',
-          required: false
-        },
-        baseUrl:{
-          help: 'The base url the component is hosted from. Default http://localhost:port/',
-          required: false
-        },
         fallbackRegistryUrl: {
-          help: 'Url to another registry which will be used by dev registry when component cannot be found in local registry. Example: --fallbackRegistryUrl=http://anotherhost:anotherport/',
-          required: false
+          description: 'Url to another registry which will be used by dev registry when component cannot be found in local registry',
+        },
+        hotReloading: {
+          boolean: true,
+          description: 'Enables hot reloading. Note: when hot reloading is set to true, each request to the component will make the registry to create a new instance for the javascript closures to be loaded, while when false the instance will be recycled between components executions',
+          default: true
         }
-      }
+      },
+      usage: 'Usage: $0 dev <dirName> [port] [baseUrl] [options]'
     },
 
     init: {
-      help: 'Creates a new empty component in the current folder',
+      cmd: 'init <componentName>',
```

Comment:
as above - if you'd like to I can walk you through differences (I've tried to include everything in the PR comment).